### ISSUE TITLE
Automatically adjust javascript field to fit contents

### ISSUE THREAD

**Issue by timstawowski** [camunda/camunda-modeler#1139]
__Is your feature request related to a problem? Please describe.__

I'm always frustrated when I have to add or recheck my scripting code in the appropriate panel section and I have to rearrange the window to show the full content.

__Describe the solution you'd like__

The field should automatically adjust to fit contents.

__Describe alternatives you've considered__

No alternatives in mind.

__Additional context__

![script-panel-1](https://user-images.githubusercontent.com/46934301/51663178-5c074500-1fb6-11e9-8a9f-23144df17eb4.png)

![script-panel-2](https://user-images.githubusercontent.com/46934301/51663188-61fd2600-1fb6-11e9-8474-d391e2ad853a.png)

**Comment by nikku** [comment]
Is this the same / duplicate of https://github.com/camunda/camunda-modeler/issues/1131 with additional details?

**Comment by timstawowski** [comment]
yes

**Comment by nikku** [comment]
Did you consider using external scripts for longer script sections? There is a few things we won't offer in the near future, such as editor tooling / syntax coloring. So our general recommendation is to not use inline scripts but external ones instead.

**Comment by timstawowski** [comment]
Okay, get that. Thought I could use these sections to create prototypically processes on the fly before I implement the ones which are more detailed. But I guess it is not a big problem switching to external scripts, so thanks!

**Comment by nikku** [comment]
Queued in backlog, regardless of our recommendation.

**Comment by nikku** [comment]
Closing this issue.

We have "fit to content" as a mechanism in place nowadays (https://github.com/bpmn-io/bpmn-js-properties-panel/pull/864) and we're happy to take contributions to further roll that out across the code base.
